hr_utilization_analysis/report/hr_utilization_analysis.py

```diff
@@ -52,17 +52,75 @@ def _compute_entry_ids(self):
         for analysis in self:
             employee_ids = HrEmployee.search(analysis._get_employees_domain())
 
-            entry_ids = [(5, False, False)]
-            for employee_id in employee_ids:
-                date = analysis.date_from
-                one_day = timedelta(days=1)
-                while date <= analysis.date_to:
-                    entry_ids.append(
-                        (0, False, {"employee_id": employee_id.id, "date": date})
+            dates = []
+            date = analysis.date_from
+            one_day = timedelta(days=1)
+            while date <= analysis.date_to:
+                dates.append(date)
+                date += one_day
+
+            analysis.entry_ids = [
+                (5, False, False),
+                *[(0, False, d) for d in self._get_entry_values(employee_ids, dates)],
+            ]
+
+    def _get_entry_values(self, employees, dates):
+        Module = self.env["ir.module.module"]
+        AccountAnalyticLine = self.env["account.analytic.line"]
+
+        project_timesheet_holidays = Module.with_user(SUPERUSER_ID).search(
+            [("name", "=", "project_timesheet_holidays"), ("state", "=", "installed")],
+            limit=1,
+        )
+
+        uom_hour = self.env.ref("uom.product_uom_hour")
+
+        all_line_ids = AccountAnalyticLine.search(
+            [
+                ("project_id", "!=", False),
+                ("employee_id", "in", employees.ids),
+                ("date", "in", dates),
+            ]
+        )
+        entries = []
+        for employee in employees:
+            tz = pytz.timezone(employee.resource_calendar_id.tz)
+            from_datetime = datetime.combine(min(dates), time.min).replace(tzinfo=tz)
+            to_datetime = datetime.combine(max(dates), time.max).replace(tzinfo=tz)
+            work_time = employee.list_work_time_per_day(from_datetime, to_datetime)
+            work_time_by_date = {w[0]: w[1] for w in work_time}
+            if project_timesheet_holidays:
+                leaves = employee.list_leaves(from_datetime, to_datetime)
+                leaves_by_date = {leave[0]: leave[1] for leave in leaves}
+
+            for date in dates:
+                line_ids = all_line_ids.filtered(
+                    lambda l: l.employee_id == employee and l.date == date
+                )
+
+                capacity = work_time_by_date.get(date, 0)
+                if project_timesheet_holidays:
+                    capacity -= leaves_by_date.get(date, 0)
```

Comment:
This is subtracting the leave twice, because the work time is already corrected for leaves [here](https://github.com/OCA/OCB/blob/13.0/addons/resource/models/resource.py#L488).